File: Practicing/Java NetBeans/ArbolBinario/src/main/java/arbolbinario/Nodo.java

package arbolbinario;

public class Nodo {
    
    private Nodo izquierda = null;
    private int data;
    private Nodo derecha = null;
    private Nodo padre = null;
    
    public Nodo(){
        
    }
    
    public Nodo(int value){
        this.data = value;
    }

    public Nodo getIzquierda() {
        return izquierda;
    }

    public void setIzquierda(Nodo izquierda) {
        this.izquierda = izquierda;
    }

    public int getData() {
        return data;
    }

    public void setData(int data) {
        this.data = data;
    }

    public Nodo getDerecha() {
        return derecha;
    }

    public void setDerecha(Nodo derecha) {
        this.derecha = derecha;
    }

    public Nodo getPadre() {
        return padre;
    }

    public void setPadre(Nodo padre) {
        this.padre = padre;
    }
    
}

File: Practicing/Python/ArbolBinario/Nodo.py
class Nodo:
    def __init__(self, value):
        self.__data = value
        self.__derecha = None
        self.__izquierda = None
    
    def setData(self, value):
        self.__data = value

    def getData(self):
        return self.__data

    def setDerecha(self, derecha):
        self.__derecha = derecha

    def getDerecha(self):
        return self.__derecha

    def setIzquierda(self, izquierda):
        self.__izquierda = izquierda

    def getIzquierda(self):
        return self.__izquierda